sections/real-life-examples/test-front-end-with-integration-back-end-with-e2e.md

```diff
@@ -109,10 +109,9 @@ You can switch focus between UI integration and E2E tests by using a conditional
 ```javascript
 // stub-services.js : a file that only includes a function to stub the back-end services
 export default function() {
-  cy.server();
   // all routes to the specified endpoint will respond with pre-packaged Json data
-  cy.route('/api/../me', 'fx:services/me.json');
```

Comment:
there was also a bug here, as of 2 weeks ago, where minimatch did not work , you had to regex it. Again, let's wait for the team to address it.

Reply:
Same here, did they fix the bug?

Reply:
Yes